packages/trace-viewer/src/ui/sourceTab.tsx

```diff
@@ -80,9 +81,24 @@ export const SourceTab: React.FunctionComponent<{
 
   const showStackFrames = (stack?.length ?? 0) > 1;
 
+  const getFileName = (fullPath?: string, lineNum?: number) => {
```

Comment:
You are creating a new function every time the component renders, consider moving this function out of the component.

Reply:
✅ fixed

Reply:
✅ fixed